module/zfs/spa.c

```diff
@@ -113,7 +113,11 @@ typedef struct zio_taskq_info {
 } zio_taskq_info_t;
 
 static const char *const zio_taskq_types[ZIO_TASKQ_TYPES] = {
+#ifdef __linux__
 	"iss", "iss_h", "int", "int_h"
+#else
+	"issue", "issue_high", "intr", "intr_high"
+#endif
```

Comment:
The change in zio.c for `zio_type_name` was the motivation. I had to go chase down why they didn't match openzfs.  I'd be happy to just make it a comment for future integrators.